packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.6+1
+
+* Remove un-used method params in webview\_flutter
```

Comment:
nit:

```suggestion
* Remove un-used method params in webview_flutter
```

Reply:
in markdown '_' needs to be escaped, or it waits for it to be closed as an underline.

Reply:
Oh, right. My mistake!